src/main/scala/uk/gov/nationalarchives/tdr/api/model/consignment/ConsignmentReference.scala

```diff
@@ -0,0 +1,19 @@
+package uk.gov.nationalarchives.tdr.api.model.consignment
+
+import uk.gov.nationalarchives.oci.Alphabet.Alphabet
+import uk.gov.nationalarchives.oci.{Alphabet, BaseCoder, IncludedAlphabet}
+
+object ConsignmentReference {
+  val alphabet: Alphabet = Alphabet.loadAlphabet(Right(IncludedAlphabet.GCRb25)) match {
+    case Left(error) => throw new Exception(s"Theres been an error creating the consignment reference", error)
```

Comment:
Just an update to the error message: it happens when the alphabet is being loaded rather than when the consignment reference is being created.

Reply:
Sorry, I think I accidentally resolved this conversation rather than one of the other ones.